src/bundler.js

```diff
@@ -71,15 +71,15 @@ export async function bundleProd({
 	cwd = cwd || '';
 	root = root || cwd;
 
-	const htmlFiles = await glob('**/*.html', {
-		cwd,
-		absolute: true,
-		filesOnly: true
-	});
-
 	// note: we intentionally pass these to Rollup as posix paths
 	const ignore = /^\.\/(node_modules|dist|build)\//;
-	const input = htmlFiles.map(p => './' + pathToPosix(relative('.', p))).filter(p => !ignore.test(p));
+	/** @type {string[]} */ const input = [];
+
+	await totalist(cwd, (rel, abs) => {
+		if (ignore.test(abs)) return;
+		if (!/\.html?/.test(rel)) return;
+		input.push('./' + pathToPosix(relative('.', abs)));
```

Comment:
should this be relative from `root` (aka, true value for `cwd`)?
```suggestion
		input.push('./' + pathToPosix(relative(root, abs)));
```

Reply:
@lukeed yes I think that's correct. The `input` should look like `['./public/index.html']`.

Reply:
Yes it looks like that either way. The question is more like do we trust `root` or not

Reply:
I usually check by running in the `public` vs `nopublic` fixtures:

```sh
# with ./public
node --experimental-modules src/cli.js build --cwd test/fixtures/simple

# no ./public
node --experimental-modules src/cli.js build --cwd test/fixtures/import-relative
```

Reply:
`root` is definitely the right way to go here because it's the CWD for rollup, to which `input` has to be relative 👍

Reply:
Both `'.'` and `root` work identically with the fixtures.

(However, unrelated, `test/fixtures/import-relative` cannot be rebuilt unless its `dist` is deleted)

Reply:
heh - yeah we should probably gitignore `dist` within the fixtures directory. Fixtures get copied to a temp dir, but we don't filter out `dist` because they're not really supposed to be run directly like I recommended :P